### DISCUSSION TITLE
I can't install kutt

### DISCUSSION THREAD

**Discussion by Doncellup** [thedevs-network/kutt#610] [Q&A] [answered]
Hello, is the project still alive?
I would like to install it, but I have errors every time… I would like to know if the project was still alive before sending the errors.
Do you have a discord server to communicate more easily?
Thanks in advance,
Doncellup

**Comment by Doncellup** [comment, marked as answer]
I use yourls.

**Comment by poeti8** [comment]
It is alive and I don't have any plans to abandon it in any way. In fact, these days I'm finding more time to work on it. 
I rather to have issues here so they can be searched and found by other people.
